nextflow_schema.json

```diff
@@ -0,0 +1,406 @@
+                },
+                "FW_primer": {
+                    "type": "string",
+                    "description": "Forward primer sequence"
+                },
+                "RV_primer": {
+                    "type": "string",
+                    "description": "Reverse primer sequence"
+                },
+                "metadata": {
+                    "type": "string",
+                    "description": "Path to metadata sheet, when missing most downstream analysis are skipped (barplots, PCoA plots, ...)."
+                },
+                "multipleSequencingRuns": {
+                    "type": "boolean",
+                    "description": "If samples were sequenced in multiple sequencing runs. Expects one subfolder per sequencing run in the folder specified by \"--input\" containing sequencing data of the specific run. These folders may not contain underscores. Also, fastQC is skipped because multiple sequencing runs might create overlapping file names that crash MultiQC."
+                },
+                "manifest": {
+                    "type": "string",
+                    "description": "Path to tab seperated table with the following labels in this exact order: sampleID, forwardReads, reverseReads. Multiple sequencing runs not supported by manifest at this stage."
+                }
+            },
+            "required": [
+                "input",
+                "FW_primer",
+                "RV_primer"
+            ],
+            "fa_icon": "fas fa-terminal"
+        },
+        "cutoffs": {
+            "title": "Cutoffs",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "retain_untrimmed": {
+                    "type": "boolean",
+                    "description": "Cutadapt will retain untrimmed reads, choose only if input reads are not expected to contain primer sequences."
+                },
+                "trunclenf": {
+                    "type": "integer",
+                    "description": "DADA2 read truncation value for forward strand"
+                },
+                "trunclenr": {
+                    "type": "integer",
+                    "description": "DADA2 read truncation value for reverse strand"
+                },
+                "trunc_qmin": {
+                    "type": "integer",
+                    "default": 25,
+                    "description": "If --trunclenf and --trunclenr are not set, these values will be automatically determined using this mean quality score"
+                }
+            },
+            "fa_icon": "fas fa-filter"
+        },
+        "taconomic_database": {
+            "title": "Taconomic database",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "reference_database": {
```

Comment:
this is not listed in the `help message

Reply:
True, but its also never used. This is not supposed to be changed. Just in case the classifier location would change (and that really shouldn't happen).

Reply:
Ok, I added it. Cannot hurt!